app/Http/Requests/ReviewsRequest.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
@sbsmith86 @katiecrane here is the new validation (instead of in the Reivews Controller!)